FBSDKCoreKit/FBSDKCoreKit/Internal/AppEvents/FBSDKUserDataStore.m

```diff
@@ -18,7 +18,7 @@
 
 #import "FBSDKUserDataStore.h"
 
-#import <FBSDKSettings.h>
+#import "FBSDKSettings.h"
```

Comment:
Here is the change needed to build.

Reply:
This has been resolved in commit e3ff562.